core/src/transaction_meta.rs

```diff
@@ -66,27 +77,23 @@ impl TransactionMeta {
         self.dead_cell.is_empty()
     }
 
-    pub fn is_new(&self) -> bool {
-        self.dead_cell.none()
+    pub fn is_dead(&self, index: usize) -> Option<bool> {
+        self.dead_cell.get(index)
     }
 
-    pub fn is_all_dead(&self) -> bool {
+    pub fn all_dead(&self) -> bool {
         self.dead_cell.all()
     }
```

Comment:
**Note**
This is wrong: because `first bit indicate if transaction is a cellbase transaction`.
So I removed this bit and add a `bool` to store whether it's a cellbase transaction.